pkg/api/rest/types.go

```diff
@@ -87,3 +87,13 @@ func (svcStrategy) AllowCreateOnUpdate() bool {
 func (svcStrategy) ValidateUpdate(ctx api.Context, obj, old runtime.Object) fielderrors.ValidationErrorList {
 	return validation.ValidateServiceUpdate(old.(*api.Service), obj.(*api.Service))
 }
+
+// ConnectRequest is an object passed to admission control for Connect operations
+type ConnectRequest struct {
+	Name     string
+	Options  runtime.Object
```

Comment:
This is not obvious to me right now, it probably will be later as I review what Options is.